packages/core-utils/src/itinerary.js

```diff
@@ -488,6 +488,30 @@ export function calculatePhysicalActivity(itinerary) {
   };
 }
 
+/**
+ * @param  {carbonIntensity} carbonIntensity carbon intensity by mode in grams/meter
+ * @param {units} units units to be used in return value
+ * @param  {itinerary} itinerary OTP trip itinierary
+ */
+export function calculateEmissions(carbonIntensity, units, itinerary) {
```

Comment:
I haven't seen the trip-details part of this yet, but do you think it's possible/makes sense to put carbon density as a constant inside of core-utils?

Reply:
Good idea, I'll provide defaults in here.